packages/services/f-braze-adapter/src/services/BrazeDispatcher.js

```diff
@@ -42,10 +42,9 @@ function interceptInAppMessagesHandler (message) {
             appboy.display.showInAppMessage(message);
         })
         .catch(error => {
-            this.logger('error', `Error handling message - ${error}`, { message, error });
+            dispatcherEventStream.publish(LOGGER, ['error', `Error handling message - ${error}`, { message, error }]);
```

Comment:
Has `'Error handling message - '` been left in by mistake?

Reply:
no its part of the message